### PR TITLE
navigation

### PR DESCRIPTION
**Pull request by sagivf**
Are we sticking to the current rethinkdb web admin ui/ux? I'm not particularly savvy with creating nice UI without exact details. 

Please check out my comments

### CODE REVIEW

File: app/renderer/views/Home/index.js [hunk 1]
```diff
@@ -16,6 +17,11 @@ function Home(props) {
         <Link to="/newConnection">
           <Icon type="plus" size={32} />
         </Link>
+        <div>
```
Comment (sagivf) [inline app/renderer/views/Home/index.js]: This it temp to be able to skip connection screen

File: app/renderer/views/Page/Footer.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import styled from 'react-emotion'
+import theme from '@/style/common'
+
+const Container = styled.footer`
```
Comment (sagivf) [inline app/renderer/views/Page/Footer.js]: I don't really like creating a separate file when there is no real necessity - it's nice having the smaller components and style next to the JSX, part of the idea of css on js I think. Not critical though.

File: app/renderer/views/Explorer/index.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import Page from '../Page/index'
```
Comment (oferitz) [inline app/renderer/views/Explorer/index.js]: No need to explicitly reference index in the import statement if index.js is present in this folder

**Comment by sagivf** [comment]
@oferitz fixed comments

**Comment by sagivf** [comment]
May I squash and merge?

**Comment by oferitz** [comment]
@sagivf sure! Nice work.

**Pull request merged**